test/tandem/testTandemSimulator.js

```diff
@@ -0,0 +1,608 @@
+    describe('normal', function() {
+      var val = {
+        time: '2014-09-25T01:00:00.000Z',
+        deviceTime: '2014-09-25T01:00:00',
+        timezoneOffset: 0,
+        conversionOffset: 0,
+        deviceId: 'InsOmn1234',
+        normal: 1.3,
+        type: 'bolus',
+        subType: 'normal'
+      };
+
+      it('passes through', function(){
+        simulator.bolus(val);
+        expect(simulator.getEvents()).deep.equals([val]);
+      });
+
+      it('does not pass through a zero-volume bolus that does not have an expectedNormal', function() {
+        var zeroBolus = _.assign({}, val, {normal: 0.0, time: '2014-09-25T01:05:00.000Z', deviceTime: '2014-09-25T01:05:00'});
+        simulator.bolus(val);
+        simulator.bolus(zeroBolus);
+        expect(simulator.getEvents()).deep.equals([val]);
+      });
+    });
+
+    describe('square', function(){
+      var val = {
+        time: '2014-09-25T01:00:00.000Z',
+        deviceTime: '2014-09-25T01:00:00',
+        timezoneOffset: 0,
+        conversionOffset: 0,
+        deviceId: 'InsOmn1234',
+        extended: 1.4,
+        duration: 1800000,
+        type: 'bolus',
+        subType: 'square'
+      };
+
+      it('passes through', function(){
+        simulator.bolus(val);
+        expect(simulator.getEvents()).deep.equals([val]);
+      });
+    });
+
+    describe('dual', function(){
+      var val = {
+        time: '2014-09-25T01:00:00.000Z',
+        deviceTime: '2014-09-25T01:00:00',
+        timezoneOffset: 0,
+        conversionOffset: 0,
+        deviceId: 'InsOmn1234',
+        normal: 1.3,
+        extended: 1.4,
+        duration: 0,
+        type: 'bolus',
+        subType: 'dual/square'
+      };
+      it('passes through', function(){
+        simulator.bolus(val);
+        expect(simulator.getEvents()).deep.equals([val]);
+      });
+    });
+  });
+
+  describe('wizard', function() {
+    var bolus = {
+      time: '2014-09-25T01:00:00.000Z',
+      deviceTime: '2014-09-25T01:00:00',
+      timezoneOffset: 0,
+      conversionOffset: 0,
+      deviceId: 'InsOmn1234',
+      normal: 1.3,
+      type: 'bolus',
+      subType: 'normal'
+    };
+
+    var val = {
+      time: '2014-09-25T01:00:00.000Z',
+      deviceTime: '2014-09-25T01:00:00',
+      timezoneOffset: 0,
+      conversionOffset: 0,
+      deviceId: 'InsOmn1234',
+      recommended: {
+        carb: 1.0,
+        correction: 2.0,
+        net: 3.0
+      },
+      bgInput: 200,
+      carbInput: 15,
+      insulinOnBoard: 0.2,
+      insulinCarbRatio: 15,
+      insulinSensitivity: 50,
+      bgTarget: {
+        target: 100,
+        high: 120
+      },
+      bolus: bolus,
+      units: 'mg/dL',
+      type: 'wizard'
+    };
+
+    it('passes through with a bolus', function() {
+      simulator.wizard(val);
+      expect(simulator.getEvents()).deep.equals([val]);
+    });
+  });
+
+  describe('deviceEvent', function() {
+  /* TODO: revisit these when all device events are implemented
```

Comment:
I'm not a huge fan of pulling in commented-out code, especially with the fixtures still OmniPod-style. (Ah, but _all_ the fixtures are OmniPod-style in this file...want to change the `deviceId` globally?)

I'd much prefer checking in pending tests (couple ways to do this with mocha - see the docs). That way we _see_ the pending tests when we run them locally.